frontend/src/atoms.tsx

```diff
@@ -293,7 +275,10 @@ export function LoadData(props: { children?: ReactNode }) {
             }
         }
         startWatch()
+
+        const timeout = setInterval(processEventQueue, 1000)
```

Comment:
Have you played with different values? Would half a second work? And maybe this should be stored in configuration?

Reply:
A half sec would work.